### ISSUE TITLE
The background used to highlight queries & db are unreadable for the colorblind.

### ISSUE THREAD

**Issue by rhuppert** [josevalim/rails-footnotes#146]
7 to 8 percent of the population is colorblind.  There is little contrast between the red background and blue foreground of those elements which you wish to emphasize, rendering them virtually unreadable to me and I suspect quite a few others.  
Would it be possible to change those to either a light background with dark fonts or vice versa?

**Comment by joshuapaling** [comment]
I'm happy to have a crack at this. Can you upload a screenshot pointing out any specific bits which need higher contrast? I'll then change some colours and upload a screenshot of the changed version here for you to check, before I submit a PR.

**Comment by Intrepidd** [comment]
Thanks for stepping in @joshuapaling 

@rhuppert Do you have suggestions regarding specific "safe" colors ?

**Comment by rhuppert** [comment]
Thanks  @joshuapaling and @Intrepidd .

Screenshot attached.  Anything with the red background is not visible to me.  Changing the foreground font color to white would do the trick or perhaps a dark (black) font color against a yellow background would also work.  I had actually modified this locally in a prior version, but did not think to do a pull request, as I thought my mod might have been incomplete.  Thanks again.

![rails-footnotes-full](https://cloud.githubusercontent.com/assets/1121284/12590296/0edd2504-c417-11e5-9f97-9373cd1064e4.png)

**Comment by joshuapaling** [comment]
Should be fixed by https://github.com/josevalim/rails-footnotes/pull/147

**Comment by rhuppert** [comment]
This may not be a rails-footnotes issue, but I realized recently I was still getting the red background.  I was running 4.1.8 according to Bundler, but after checking the gem in my rbenv folder, I realized the affected files had not updated from 4.1.7.  I resolved it by running gem uninstall for r-f and then doing a bundle update.
Am I missing something?  As I implied, this may be a bundler or possibly a rbenv issue, but I thought I'd ask here first since this gem is where I noticed this behavior.
FYI, Ruby 2.3.0, Rails 4.2.6. Bundler 1.11.2, Rbenv 1.0.0-14-gc388331.

And thank you both @joshuapaling and @Intrepidd regarding the fixed issue.
